Source/WebKit/UIProcess/ApplicationStateTracker.mm

```diff
@@ -78,6 +80,9 @@ - (id)initWithParent:(WebKit::ApplicationStateTracker&)parent
 
 - (void)setObservedWindow:(UIWindow *)window
 {
+    if (!linkedOnOrAfterSDKWithBehavior(SDKAlignedBehavior::ApplicationStateTrackerDoesNotObserveWindow))
```

Comment:
It would be ideal if we could detect the use of Aspects.framework and generate some kind of log message here. But that is likely to be hard/stupid/fragile, so may cause more trouble than its worth.